src/labels/CustomLabels.labels

```diff
@@ -5705,7 +5705,7 @@ IMPORTANT: Both of these fields must have the same data type.</value>
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>Help text for Contact Role for Organizational Opps</shortDescription>
-        <value>The default Contact Role automatically assigned to the Opportunity's Primary Contact when the Opportunity's Account is an Organization Account.</value>
+        <value>The default Contact Role automatically assigned to the Opportunity's Primary Contact when the Opportunity's Account is NOT a Household, One-to-One, or Individual Account.</value>
```

Comment:
Single quotes should be encoded in custom labels. You should be able to get the encoded version by refreshing the 'custom.label' file from within your ide.